src/bp-templates/bp-nouveau/js/buddypress-media.js

```diff
@@ -6414,7 +6413,7 @@ window.bp = window.bp || {};
 			
 			$('.bb-media-model-wrapper.media .bb-media-section').find('img').attr('src', '');
 			if( $('.bb-media-model-wrapper.video .bb-media-section').find('video').length ){
-				$('.bb-media-model-wrapper.video .bb-media-section').find('video').attr('src', '');
+				videojs( $('.bb-media-model-wrapper.video .bb-media-section').find('video').attr('id') ).reset();
```

Comment:
@bb-yudhisthir Need to define videojs as global at top of the file.
Please check line number 2 in this file - src/bp-templates/bp-nouveau/js/buddypress-video.js